cli/flox/src/commands/environment.rs

```diff
@@ -158,11 +158,11 @@ impl Edit {
 
                             Please `exit` the environment and run `flox activate` to see these changes."});
                 } else {
-                    println!("✅ environment successfully edited");
+                    println!("✅  Environment successfully updated.");
```

Comment:
accidentally added an extra space here

Reply:
or is this intentional to account for the icon?

Reply:
Yeah that's deliberate, we want that after every emoji